### PR TITLE
issue 448.1: update documentation

### PR DESCRIPTION
**Pull request by vasiliy-arangodb**

### CODE REVIEW

File: Documentation/Books/Manual/Views/ArangoSearch/README.md [hunk 1]
```diff
@@ -1,54 +1,293 @@
...
+
+- BM25
+  order results based on the
+  [BM25 algorithm](https://en.wikipedia.org/wiki/Okapi_BM25)
+
+- TFIDF
+  order results based on the
+  [TFIDF algorithm](https://en.wikipedia.org/wiki/TF-IDF)
+
+### ArangoSearch is much more than a fulltext search
```
Comment (graetzer) [inline Documentation/Books/Manual/Views/ArangoSearch/README.md]: I don't really get the point of this subsection, what is this supposed to tell me ?

File: Documentation/Books/Manual/Views/ArangoSearch/README.md [hunk 2]
```diff
@@ -1,54 +1,293 @@
...
+The IResearch functionality is exposed to ArangoDB via the view API for views
+of type *arangosearch*, because the ArangoSearch view is merely an identity
+transformation applied onto documents stored in linked collections of the same
+ArangoDB database.
+In plain terms an ArangoSearch view only allows searching and sorting of documents
+located in collections of the same database.
+The matching documents themselves are returned as-is from their corresponding collections.
+
+### Links to ArangoDB collections
+
```
Comment (graetzer) [inline Documentation/Books/Manual/Views/ArangoSearch/README.md]: we should mention here that links do appear as Indexes in collections

File: Documentation/Books/Manual/Views/ArangoSearch/README.md [hunk 3]
```diff
@@ -1,54 +1,293 @@
...
+ArangoDB collections and similarly any ArangoDB collection may be referenced by
+multiple ArangoSearch views.
+
+To configure an ArangoSearch view for consideration of documents from a given
+ArangoDB collection a link definition must be added to the properties of the
+said ArangoSearch view defining the link parameters as per the section
+[View definition/modification](#view-definitionmodification).
+
+### Analyzers
+
```
Comment (graetzer) [inline Documentation/Books/Manual/Views/ArangoSearch/README.md]: is it actually possible to add custom analyzers now ?

**Comment by vasiliy-arangodb** [comment]
replaced by https://github.com/arangodb/arangodb/pull/6215

**Pull request closed without merging**